config/initializers/session_store.rb

```diff
@@ -1 +1,15 @@
-Rails.application.config.session_store :active_record_store, key: '_upaya_session'
+require 'idle_session_expiration_store'
+
+options = {
+  key: '_upaya_session',
+  redis: {
+    driver: :hiredis,
+    expire_after: Figaro.env.session_timeout_in_minutes.to_i,
```

Comment:
I just updated my PR to use `Devise.timeout_in` so that we are dealing with an integer rather than a string. Should we do that here, too? https://github.com/18F/identity-idp/pull/596/files